soh/soh/OTRGlobals.cpp

```diff
@@ -1423,3 +1445,138 @@ extern "C" void* getN64WeirdFrame(s32 i) {
     char* weirdFrameBytes = reinterpret_cast<char*>(n64WeirdFrames);
     return &weirdFrameBytes[i + sizeof(n64WeirdFrames)];
 }
+
+extern "C" s16 GetItemModelFromId(s16 itemId) {
```

Comment:
I think it might be useful to prefix randomizer specific methods. If they're more generally useful, they should be moved out of the Randomizer class.

We have these examples:
```c++
OTRGetAspectRatio()
ResourceMgr_LoadCSByName
AudioPlayer_GetDesiredBuffered()

//
Randomizer_GetItemModelFromId()
Randomizer_LoadSettings()
```